app/src/components/map/components/ImportBoundaryDialog.tsx

```diff
@@ -0,0 +1,36 @@
+import Alert from '@mui/material/Alert';
+import Box from '@mui/material/Box';
+import ComponentDialog from 'components/dialog/ComponentDialog';
+import FileUpload from 'components/file-upload/FileUpload';
+import { ProjectSurveyAttachmentValidExtensions } from 'constants/attachments';
+import { Feature } from 'geojson';
+import { boundaryUploadHelper } from 'utils/mapBoundaryUploadHelpers';
+
+export interface IImportBoundaryDialogProps {
+  isOpen: boolean;
+  onClose: () => void;
+  onSuccess: (features: Feature[]) => void;
+  onFailure: (message: string) => void;
+}
+
+export const ImportBoundaryDialog = (props: IImportBoundaryDialogProps) => {
```

Comment:
Nit: prefer default exports for React components

Reply:
updated